native/src/main/java/io/ballerina/persist/components/Function.java

```diff
@@ -0,0 +1,155 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.persist.components;
+
+import io.ballerina.compiler.syntax.tree.AbstractNodeFactory;
+import io.ballerina.compiler.syntax.tree.AnnotationNode;
+import io.ballerina.compiler.syntax.tree.FunctionBodyNode;
+import io.ballerina.compiler.syntax.tree.FunctionDefinitionNode;
+import io.ballerina.compiler.syntax.tree.FunctionSignatureNode;
+import io.ballerina.compiler.syntax.tree.IdentifierToken;
+import io.ballerina.compiler.syntax.tree.IfElseStatementNode;
+import io.ballerina.compiler.syntax.tree.Node;
+import io.ballerina.compiler.syntax.tree.NodeFactory;
+import io.ballerina.compiler.syntax.tree.NodeList;
+import io.ballerina.compiler.syntax.tree.ReturnTypeDescriptorNode;
+import io.ballerina.compiler.syntax.tree.StatementNode;
+import io.ballerina.compiler.syntax.tree.SyntaxKind;
+import io.ballerina.compiler.syntax.tree.Token;
+import io.ballerina.compiler.syntax.tree.TypeDescriptorNode;
+import io.ballerina.persist.nodegenerator.SyntaxTreeConstants;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.ballerina.persist.components.TypeDescriptor.getReturnTypeDescriptorNode;
+
+/**
+ * Class representing FunctionDefinitionNode.
+ *
+ @since 0.1.0
+ */
+public class Function {
+
+    private final SyntaxKind kind = SyntaxKind.OBJECT_METHOD_DEFINITION;
+    private NodeList<Token> qualifierList;
+    private final Token finalKeyWord = AbstractNodeFactory.createIdentifierToken("function ");
```

Comment:
Move to constants.

Reply:
updated in [6d8b4a4](https://github.com/ballerina-platform/persist-tools/pull/15/commits/6d8b4a41b9a0ea1002bb906bb2008380cfa7cdb1)